### PR TITLE
Attempted to flesh out the "governance" documents.

### PR DESCRIPTION
**Pull request by rbeyer**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+Contributions are welcome, and they are greatly appreciated! Every
+little bit helps, and credit will always be given.
+
+You can contribute in many ways:
+
+## Types of Contributions
+
+### Contribute your script
+
+We want it to be easy to get that useful script of yours into the Scriptorium
```
Comment (AndrewAnnex) [inline CONTRIBUTING.md]: I think maybe this sentence belongs at the top, as a better introduction to the contributing header

File: README.md [hunk 2]
```diff
@@ -1,2 +1,31 @@
-# scriptorium
-A collection of scripts
+# Scriptorium
+
+This PlanetaryPy repository is not a focused library, but a collection
+of scripts.  You may find something here useful, and if so, feel
```
Comment (AndrewAnnex) [inline README.md]: maybe add a bit more here, e.g.: "scripts useful for Planetary Scientists." And move the concept paragraph up. I think the download instructions could be moved down to their own subsection

**Review by AndrewAnnex — APPROVED** [review]
looks good, I left a few comments to shuffle words around for better flow but there aren't strictly necessary

File: CONTRIBUTING.md (hunk 1 above)
Comment (rbeyer) [reply CONTRIBUTING.md]: Moved.

File: README.md (hunk 2 above)
Comment (rbeyer) [reply README.md]: So I think there will be many more folks coming to "use" a script than there will be who will "deposit" a script, so I wanted to have the "download" instructions close to the top.  I've broken things up into sections, but did leave the "Use a Script" instructions at the top.

**Pull request merged**